client/app.jsx

```diff
@@ -72,17 +78,18 @@ export default class App extends React.Component {
 }
 
 App.contextType = AppContext;
+Home.contextType = AppContext;
 Decks.contextType = AppContext;
+Header.contextType = AppContext;
+Header.contextType = AppContext;
+AddCard.contextType = AppContext;
+NewDeck.contextType = AppContext;
 AuthForm.contextType = AppContext;
 NotFound.contextType = AppContext;
-Header.contextType = AppContext;
 EditCard.contextType = AppContext;
-Home.contextType = AppContext;
-AddCard.contextType = AppContext;
-StudyCards.contextType = AppContext;
 ViewCards.contextType = AppContext;
-NewDeck.contextType = AppContext;
+CardStudy.contextType = AppContext;
```

Comment:
It's best to put these context assignments in the modules of these components, so that the dependency is explicit

Reply:
Got it. I moved everything like this where each module imports the context and I set the context. This cleared up a lot of the imports and bottom clutter of app.jsx

![image](https://user-images.githubusercontent.com/109567499/202512000-527d6a6c-af07-43e0-a402-31f4353d2d2c.png)